examples/int8/training/vgg16/test_qat.py

```diff
@@ -81,7 +81,7 @@ def test(model, dataloader, crit):
 import trtorch
 # trtorch.logging.set_reportable_log_level(trtorch.logging.Level.Debug)
 compile_settings = {
-"input_shapes": [[1, 3, 32, 32]],
+"inputs": [trtorch.Input([1, 3, 32, 32])],
 "op_precision": torch.int8 # Run with FP16

```

Comment:
We can change this to `enabled_precisions: {torch.int8}`. I can do this in a separate PR if you want to merge this quick.

Reply:
ill just add it to this one